snippets/distr-tracing-tempo-tempostack-custom-resource.adoc

```diff
@@ -4,40 +4,49 @@
 // * modules/distr-tracing-tempo-install-tempostack-web-console.adoc
 // * modules/distr-tracing-tempo-install-tempostack-cli.adoc
 
+.Example `TempoStack` CR for AWS S3 and MinIO storage and two tenants
```

Comment:
@rubenvp8510, do we have everything that we need here?
I'm looking at this task in the [Jira](https://issues.redhat.com/browse/OBSDOCS-1676)
```The OTEL deployment should include all recommended and required components - RED metrics, k8sattributes processor```